src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +120,28 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            foreach(var item in serviceNameSpace.Items) {
+                if (item.Metadata.Annotations != null) {
+                    if (item.Metadata.Annotations.TryGetValue("bridgetokubernetes/ignore-ports", out var ports)) {
+                        var allPorts = ports.Split(",");
+                        foreach(var port in allPorts) {
+                            try {
+                                portsToIgnore.Add(int.Parse(port));
+                            } catch {
+                                throw new Exception("The ports are not in the correct format");

```

Comment:
Let's add in this message the ports string user used as well as well as some guidance on how to fix. Something like:
("bridgetokubernetes/ignore-ports configuration input with value {1} is invalid. It should be a comma separated list of integer ports")

I also think it should be an user visible exception: UserVisibleException, it may require you to add the string format to Resources. You can use line 113 as an example.